app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
```

Comment:
Code Style: It's recommended to use more descriptive variable names. For example, instead of `parts`, you could use `command_parts` or `command_segments`. This will make your code easier to read and understand.